components/builder-depot/src/error.rs

```diff
@@ -29,6 +29,7 @@ use redis;
 pub enum Error {
     BadPort(String),
     DataStore(dbcache::Error),
+    Generic(String),
```

Comment:
This should be specific errors for each case. This allows you to match on errors, vs having to inspect:

```rust
return Err(Error::PackageExists)
```

Then you can use the traits for Display and Describe to make the errors clean.